src/js/mep-player.js

```diff
@@ -249,6 +252,11 @@
 
 		// unique ID
 		t.id = 'mep_' + mejs.mepIndex++;
+ 
```

Comment:
Is there a reason why this is here in the pre-init instead of somewhere like 355 where it's built? I'd like to move it there so it's together, but I wanted to double check first.